packages/workflow/src/internals.ts

```diff
@@ -304,12 +306,15 @@ export class Activator implements ActivationHandler {
   /**
    * Patches we know the status of for this workflow, as in {@link patched}
    */
-  public readonly knownPresentPatches = new Set<string>();
+  private readonly knownPresentPatches = new Set<string>();
 
   /**
    * Patches we sent to core {@link patched}
    */
-  public readonly sentPatches = new Set<string>();
+  private readonly sentPatches = new Set<string>();
+
+  private readonly knownFlags = new Set<number>();
+  private readonly newlyUsedFlags = new Set<number>();
```

Comment:
You don't need to figure out what's newly used or not, Core does that for you. You can just blindly send anything you used in this activation